modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -665,6 +665,11 @@ TEST_P(Test_ONNX_layers, Split_EltwiseMax)
     testONNXModels("split_max");
 }
 
+TEST_P(Test_ONNX_layers, LSTM_Activations)
+{
+    testONNXModels("lstm_cntk_tanh", pb, 0, 0, false, false);
```

Comment:
Please put test files into opencv_extra PR directly. "MIT License" of ONNXRuntime is compatible with OpenCV.

Downloading should be used to fetch large model files (>5Mb).